Sources/MapboxCoreNavigation/EHorizon/ObjectType.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import MapboxDirections
+import MapboxNavigationNative
+
+extension RoadObject {
+    
+    /**
+     Type of the road object.
+     */
+    public enum ObjectType {
```

Comment:
Even though it's possible to use enum with `Type` name, it'll be impossible to use initializers of that enum. That's the reason I've used `ObjectType` name here.

Reply:
Oh, good point. That’s why we’ve always called it “Kind” instead. There are some related “Kind” types, so we should try to be consistent with `RoadObject.Kind`. Otherwise, `RoadObject.ObjectType` is more redundant than what we started out with.

Reply:
Renamed `RoadObject.ObjectType` to `RoadObject.Kind`.